app/purchasing.cpp

```diff
@@ -9,6 +9,7 @@
 
 #include  <QDebug>
 #include <QNetworkReply>
+#include <QUrlQuery>
```

Comment:
remove?

Reply:
`QUrlQuery` is used by this class. I had to add it there otherwise it would not compile. It seems that it found this object before via hierarchy of includes coming from inpututils->positionkit->..., however when I removed positionkit from utils, few places needed new imports.